src/LanguageServer/LoggingModule.cs

```diff
@@ -72,7 +72,7 @@ static ILogger CreateLogger(IComponentContext componentContext)
             LoggerConfiguration loggerConfiguration = CreateDefaultLoggerConfiguration(languageServerConfiguration)
                 .WriteTo.LanguageServer(languageServer, languageServerConfiguration.Logging.LevelSwitch);
 
-            ILogger logger = loggerConfiguration.CreateLogger();
+            var logger = loggerConfiguration.CreateLogger();
```

Comment:
Sorry but this is one of the few scenarios where I really do have a problem with the `var` keyword (`var foo = MakeFoo()`).

Reply:
This was flagged by one of built-in analyzers. `CreateLogger` returns more concrete type than `ILogger`, thus it is better to not upcast it to an interface. Will change to a concrete type and make PR with `var` prefferences later